.github/file-paths.yaml

```diff
@@ -14,8 +14,8 @@ shared_specs: &shared_specs
 
 frontend_sources: &frontend_sources
   - *shared_sources
-  - "enterprise/frontend/**"
-  - "frontend/!(test)/**"
+  - "enterprise/frontend/!(test)**/**/{!(*.spec),}.{js,jsx,ts,tsx}"
+  - "frontend/!(test)**/**/{!(*.spec),}.{js,jsx,ts,tsx}"
```

Comment:
Alternatively, we can use `?(enterprise|frontend)/!(test)**/**/{!(*.spec),}.{js,jsx,ts,tsx}`

I like it even more, but IIRC someone asked me to keep paths separate before.

Reply:
We need to include CSS too, otherwise looks good

Reply:
@metabase/frontend-developers WDYT about:
1. using the unified glob instead of the two distinctive ones?
2. using the wildcard at the end instead of enumerating all file types?

Reply:
1. any way works for me
2. good point, I'm fine with current approach, but wildcard could be more flexible